Gruntfile.js

```diff
@@ -87,7 +87,11 @@ module.exports = function(grunt) {
         options: {
           jshintrc: '.jshintrc'
         },
-        src: 'src/js/**/*.js'
+        src: [
+          'src/js/**/*.js',
+          '!src/js/extensions/jquery.js',
+          '!src/js/extensions/jquery.once.js'
```

Comment:
Also, I'm not sure, but it would make sense to add these scripts via the grunt build process maybe? Or some how installed them via bower maybe? as part of the `grunt install` task? Idk, just thinking here. This comment is less important really.